sdk/include/opentelemetry/sdk/metrics/meter.h

```diff
@@ -148,6 +149,12 @@ class Meter final : public opentelemetry::metrics::Meter
           storage_registry_[instrument_descriptor.name_] = storage;
           return true;
         });
+    if (!success)
```

Comment:
nit - I think control will never reach here because View API will create the default view if no matching view is found. But no harm in keeping it, as the View API can change in the future.